qemu/tests/cfg/virtiofs_another_user_access.cfg

```diff
@@ -0,0 +1,52 @@
+- virtiofs_another_user_access:
```

Comment:
How about using 'virtio_fs_group_permission_access', it's more like the case described.